cmd/nsm-init/nsm-init.go

```diff
@@ -79,25 +204,10 @@ func main() {
 	logrus.Infof("nsm client: client api %+v", nsmClient)
 	// Init related activities start here
 
-	currentNamespace, err := netns.Get()
-	if err != nil {
-		logrus.Errorf("nsm client: failure to get pod's namespace with error: %+v", err)
+	if err := applyRequiredConfig(ns); err != nil {
+		logrus.Infof("nsm client: initialization failed, exiting...", err)
 		os.Exit(1)
 	}
-	logrus.Infof("nsm client: pod's namespace is [%s]", currentNamespace.String())
-	namespaceHandle, err := netlink.NewHandleAt(currentNamespace)
-	if err != nil {
-		logrus.Errorf("nsm client: failure to get pod's handle with error: %+v", err)
-		os.Exit(1)
-	}
-	interfaces, err := namespaceHandle.LinkList()
-	if err != nil {
-		logrus.Errorf("nsm client: pailure to get pod's interfaces with error: %+v", err)
-	}
-	logrus.Info("nsm client: pod's interfaces:")
-	for _, intf := range interfaces {
-		logrus.Infof("Name: %s Type: %s", intf.Attrs().Name, intf.Type())
-	}
 	// Init related activities ends here
 	logrus.Info("nsm client: initialization is completed successfully, exiting...")
 	os.Exit(0)
```

Comment:
No need for os.Exit here, 0 return is implied.

Reply:
make sense, thanks for catching it.